Sample.BasicComConnector/NewSimClient.cs

```diff
@@ -4,13 +4,15 @@
 using CogniteSdk.Alpha;
 
 using Microsoft.Extensions.Logging;
+using System.IO.Compression;
 
 public class NewSimClient : AutomationClient, ISimulatorClient<DefaultModelFilestate, SimulatorRoutineRevision>
 {
     private readonly SemaphoreSlim semaphore = new SemaphoreSlim(1, 1);
 
     private readonly string _version = "N/A";
     private readonly ILogger logger;
+    private readonly List<string> _extractedDirectories = new();
```

Comment:
why do you want to store a list of extracted Directories? (if its just for cleanup, check out my comment further on, you could perform cleanup at a different location)